caveclient/chunkedgraph.py

```diff
@@ -306,14 +319,33 @@ def get_user_operations(
         include_undo: bool = True,
         timestamp_end: datetime.datetime = None,
     ):
-        """get operation details for a user_id
+        """Get operation details for a user ID.
+
+        Parameters
+        ----------
+        user_id : int
+            User ID to query (use 0 for all users (admin only)).
+        timestamp_start : datetime.datetime, optional
+            Timestamp to start filter (UTC).
+        include_undo : bool, optional
+            Whether to include undos. Defaults to True.
+        timestamp_end : datetime.datetime, optional
+            Timestamp to end filter (UTC). Defaults to now.
 
-        Args:
-            user_id (int): userID to query (use 0 for all users [admin only])
-            timestamp_start (datetime.datetime, optional): timestamp to start filter (UTC).
-            include_undo (bool, optional): whether to include undos. Defaults to True.
-            timestamp_end (datetime.datetime, optional): timestamp to end filter (UTC). Defaults to now.
+        Returns
+        -------
+        pd.DataFrame
+            DataFrame with the following columns:
+
+            "operation_id": int
+                Identifier for the operation.
+            "timestamp": datetime.datetime
+                Timestamp of the operation.
+            "user_id": int
+                User who performed the operation.
+            # TODO not actually sure what the rest of the outputs are, not an admin
```

Comment:
this command didn't work for me (even with a non-zero `user_id`, so an admin would need to fill this out.

perhaps also worth clarifying who can use this function at all?

Reply:
good point

Reply:
Bethany also had the same issue as me - I could open a new issue if this is meant to be available to everyone as the docs currently indicate, or if not, we could clarify here. 

Although, if user info is not meant to be accessible to everyone, this info is available in the tabular changelog, so there's already a clunky workaround

Reply:
current user of the clunky workaraound

Reply:
We were conservative in what we made available to users to easily follow other peoples work. I think a discussion can be had whether this should be behind admin privileges. For giving the necessary privileges: It is easy to add dataset-specific admins, so it would be straightforward to add the two of you as that (and we should do that). While the data is available in tabular changelogs, one would still have to iterate through all neurons to trace another users actions.

Reply:
that's fair. for the purposes of this PR, I just added a note to the documentation indicating that this is only supported for admins.

separately, yes, I'd be happy to 1) get access but more importantly 2) talk about opening up this function. I don't care too much, but since there's already a roundabout way to get that info it seems like it would make sense to support for any user (or closing that loophole if not).